data_steward/bq_utils.py

```diff
@@ -320,6 +320,15 @@ def job_status_done(job_id):
     return job_running_status == 'DONE'
 
 
+def job_status_errored(job_id):
```

Comment:
Handy function.  Please provide a pep-8 style documentation string.  https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

Reply:
Added the docstring